app/components/Utility/AmountSelectorStyleGuide.jsx

```diff
@@ -125,7 +126,7 @@ class AmountSelector extends DecimalChecker {
                         onChange={this._onChange.bind(this)}
                         tabIndex={this.props.tabIndex}
                         onPaste={this.props.onPaste || this.onPaste.bind(this)}
-                        onKeyPress={this.onKeyPress.bind(this)}
+                        // onKeyPress={this.onKeyPress.bind(this)}
```

Comment:
Why?

Reply:
It was causing a bug while writing numbers in modals.

Reply:
Found the bug and fixed, I will remove the comment from here and add fix somewhere else.